get_together/templates/get_together/events/change_team.html

```diff
@@ -1,7 +1,7 @@
 {% extends "get_together/base.html" %}
 {% load static i18n %}
 {% block content %}
-<h2>{% blocktrans with event_name=event.name %}Change host for: {{event_name}}{% endblocktrans %}</h2>
+<h2>{{ theme.text_labels.change_host_for }} {{event.name}} </h2>
```

Comment:
For consistency, theme text should either always include the colon, or never include it. I'm inclined to say it should never include it, in which case it would need to be add here after the text insertion.